providers/postmark/src/lib/postmark.provider.spec.ts

```diff
@@ -44,6 +67,26 @@ test('should trigger postmark correctly', async () => {
   });
 });
 
+test('should get message ID', () => {
+  const provider = new PostmarkEmailProvider(mockConfig);
+  expect(provider.getMessageId(mockMessage)).toEqual([
+    '883953f4-6105-42a2-a16a-77a8eac79483',
+  ]);
+});
+
+test('should parse postmark webhook', () => {
+  const provider = new PostmarkEmailProvider(mockConfig);
+  const identifier = '883953f4-6105-42a2-a16a-77a8eac79483';
+  const date = new Date().toISOString();
+  expect(provider.parseEventBody(mockWebHook, identifier)).toEqual({
+    status: 'delivered',
+    date: date,
+    externalId: '883953f4-6105-42a2-a16a-77a8eac79483',
+    attempts: 1,
+    response: '',
+    row: mockWebHook,
+});
+
```

Comment:
Seems to be missing }); here

Reply:
Thanks so much for pointing out to me where the missing closing brackets and parenthesis are.